cmd/context/selector.go

```diff
@@ -34,7 +34,7 @@ func getContextsSelection(ctxOptions *ContextOptions) []utils.SelectorItem {
 	}
 	clusters := make([]utils.SelectorItem, 0)
 
-	clusters = append(clusters, utils.SelectorItem{Name: okteto.CloudURL, Label: cloudOption, Enable: true, IsOkteto: true})
+	clusters = append(clusters, utils.SelectorItem{Name: okteto.CloudURL, Label: cloudOption, Enable: true})
 	clusters = append(clusters, getOktetoClusters(true)...)
 	if len(k8sClusters) > 0 {
```

Comment:
nit: we could remove the if since `getK8sClusters` should work with empty slices

Reply:
i would like to keep the changes as tight as possible to the refactor purpose, thnks!